src/sequence-steps.js

```diff
@@ -117,15 +193,31 @@ ${ chalk.green( options.log ) }` );
 }
 
 export function askSemverJump( [ git, options ] ) {
+	const releaseChoices = [
```

Comment:
can you see if there is a `release` property on `options`?

if so, we may be able to punt out of this function.

scenario, `tag-release -p -r preminor -i robert` should skip many questions and go right to the question "Would you like to edit your log?".

this apparently was broken in a previous release, so this would be fixing something broke and also  making it to work with the new pre-release stuff.